NFT/tests/functions/meta_data.rs

```diff
@@ -1,12 +1,14 @@
 use crate::utils::{
     abi_calls::{constructor, meta_data, mint},
     test_helpers::setup,
-    Identity, TokenMetaData,
+    TokenMetaData,
 };
-use fuels::signers::Signer;
+use fuels::{prelude::Identity, signers::Signer};
 
 mod success {
 
+    use fuels::prelude::SizedAsciiString;
```

Comment:
What is this ugly type?

Reply:
That's the only way I found to pass in a static sway string.
```
TokenMetaData {
      name: SizedAsciiString::<7>::new("Example".to_string()).unwrap()
}
```